Source/AssistantV1/Models/DialogSuggestionResponseGeneric.swift

```diff
@@ -0,0 +1,181 @@
+/**
+ * (C) Copyright IBM Corp. 2019.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+
+/** DialogSuggestionResponseGeneric. */
+public struct DialogSuggestionResponseGeneric: Codable, Equatable {
+
+    /**
+     The type of response returned by the dialog node. The specified response type must be supported by the client
+     application or channel.
+     **Note:** The **suggestion** response type is part of the disambiguation feature, which is only available for Plus
+     and Premium users. The **search_skill** response type is available only for Plus and Premium users, and is used
+     only by the v2 runtime API.
+     */
+    public enum ResponseType: String {
+        case text = "text"
+        case pause = "pause"
+        case image = "image"
+        case option = "option"
+        case connectToAgent = "connect_to_agent"
+        case searchSkill = "search_skill"
+    }
+
+    /**
+     The preferred type of control to display.
+     */
+    public enum Preference: String {
+        case dropdown = "dropdown"
+        case button = "button"
+    }
+
+    /**
+     The type of response returned by the dialog node. The specified response type must be supported by the client
+     application or channel.
+     **Note:** The **suggestion** response type is part of the disambiguation feature, which is only available for Plus
+     and Premium users. The **search_skill** response type is available only for Plus and Premium users, and is used
+     only by the v2 runtime API.
+     */
+    public var responseType: String
+
+    /**
+     The text of the response.
+     */
+    public var text: String?
+
+    /**
+     How long to pause, in milliseconds.
+     */
+    public var time: Int?
```

Comment:
This is a terrible name.  I know this comes from the API doc, but I wonder if it is still possible to change it.

Reply:
@mkistler I don't think we are going to be able to change this for now. We should discuss with the docs team.